javaslang/src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -2792,6 +2792,7 @@ public int parseInt(int radix) {
      * @throws NumberFormatException If this {@code CharSeq} does not contain a parsable unsigned int.
      * @since 2.1.0
      */
+    @GwtIncompatible
     public int parseUnsignedInt() {
         return Integer.parseUnsignedInt(back);
```

Comment:
I'm not sure about this. 
In fact I'm not sure that this method is useful in Java too :smile: